tuf/keys.py

```diff
@@ -86,7 +86,9 @@
 # default.  https://github.com/pyca/ed25519
 _available_crypto_libraries = ['ed25519']
 
-# Import the PyCrypto library so that RSA keys are supported.
+# Try to import TUF's PyCrypto module (pycrypto_keys.py), and verify and record
+# that the PyCrypto library is available.  pycrypto_keys.py is used here for
+# general-purpose cryptography and RSA.
 try:
   import Crypto
```

Comment:
This import is unnecessary and adds nothing. Ditto line 103 below (import cryptography). [Albert's comment](https://github.com/theupdateframework/tuf/pull/289/files#r48359877) on PR #289 likely was trying to make this point.
 
0. This file per se does not use Crypto.
1. Importing tuf.pycrypto_keys should (and does, I think - will test) import whatever it, in turn, needs.
2. If importing tuf.pycrypto_keys fails, then we already know that we cannot use it and thereby will not be supporting Crypto.
3. This also doesn't help the user figure out what's missing because they're not seeing any messages if either import fails (which is fine).

So nothing is added.